### DISCUSSION TITLE
Regarding IGV issue consultation

### DISCUSSION THREAD

**Discussion by wuguosssssfd** [community/community#119939] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When using the latest version of IGV, I encountered an issue where the software view cannot be zoomed out to the minimum. When I try to minimize the view, I face difficulties. Our input files are in BAM format, and the window size can only be minimized to 27kb and cannot be further reduced to above 40kb, as shown in the screenshot below. Adjusting the window tracks does not resolve this issue, which is inconvenient for my data analysis and visualization.

I would like to inquire whether this issue is due to a limitation of the software or if it is an error in my configuration or usage. If it is a software issue, have you already identified this problem and addressed it in future updates? Are there better solutions available? Thank you for your assistance.
![e004a2471285361d4c25b0c6b9bc8ef3](https://github.com/community/community/assets/167315055/aa9318f6-87ab-4e04-b9f6-d0edcd34e0c5)
![fb2c796ad77d0fb7cfa4c017ad0e7f80](https://github.com/community/community/assets/167315055/1de77f9f-5029-4cbd-bc50-b94d44200f0a)

**Comment by ettaboyle** [comment]
Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
